src/utils/processUpload.js

```diff
@@ -25,9 +25,70 @@ const putInS3 = (bucketKey, loadedFile, dispatch, sample, file) => (
   )
 );
 
+const metadataFor = (bundle) => {
```

Comment:
As this function is specifically to be used for bundles, maybe you can use a better function name e.g. `metadataForBundle`?